docs/en/data_structure/keypoints.md

```diff
@@ -0,0 +1,181 @@
+# Keypoints
+
+- [Overview](#overview)
+- [Key/Value definition](#keyvalue-definition)
+- [Attribute definition](#attribute-definition)
+- [Name convention](#name-convention)
+- [Create an instance](#create-an-instance)
+- [Auto-completion](#auto-completion)
+- [Convert between numpy and torch](#convert-between-numpy-and-torch)
+- [File IO](#file-io)
+- [Keypoints convention](#keypoints-convention)
+
+### Overview
+
+Keypoints is a class for multi-frame, multi-person keypoints data, based on python dict class. It accepts either `numpy.ndarray` or `torch.Tensor`, keeps them in their original type, and offers type conversion methods.
+
+### Key/Value definition
+
+- keypoints: A tensor or ndarray for keypoints, with confidence at the last dim.
+
+  ​                kps2d in shape [n_frame, n_person, n_kps, 3],
+
+  ​                kps3d in shape [n_frame, n_person, n_kps, 4].
+
+- mask:  A tensor or ndarray for keypoint mask,
+
+  ​                in shape [n_frame, n_person, n_kps], in dtype uint8.
+
+- convention:  Convention name of the keypoints, type str,
+
+  ​                can be found in KEYPOINTS_FACTORY.
+
+We allow you to set other keys and values in a Keypoints instance, but they will be dropped when calling `to_tensor` or `to_numpy`.
+
+### Attribute definition
+
+- logger: Logger for logging. If None, root logger will be selected.
+- dtype:  The data type of this Keypoints instance, it could be one among `numpy`, `torch` or `auto`. Values will be converted to the certain dtype when setting. If dtype==auto, it will be changed the first time  `set_keypoints()` is called, and never changes.
+
+### Name convention
+
+- kps: `kps` is the abbreviation for keypoints. We use `kps` for array-like keypoints data. More precisely, we could use `kps_arr` or `kps_np` for ndarray type keypoints data, and `kps_tensor` for Tensor type data.
+- keypoints:  `keypoints` denotes an instance of `class Keypoints`, including kps data, mask and convention.
+
+### Create an instance
+
+a. Call `Keypoints()`, keypoints, mask and convention are necessary.
+
+```python
+from xrprimer.data_structure.keypoints import Keypoints
```

Comment:
Is it possible to do `from xrprimer.data_structure import Keypoints, Limbs`? I still think it may be more clean to not write keypoints or limbs twice.

Reply:
Fixed in 593876b.